frappe/core/doctype/test_field_order_doctype/test_field_order_doctype.js

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2021, Frappe Technologies and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Test Field Order DocType', {
```

Comment:
are you using this doctype?

Reply:
No, these look like they were inserted by some tests I was running. Will clean these up